native/src/constants/endpoint.ts

```diff
@@ -1,2 +1,7 @@
+import { Platform } from 'react-native'
+import NativeConstants from './NativeConstants'
+
 export const tunewsApiUrl = 'https://tunews.integreat-app.de'
 export const tunewsWebsiteUrl = 'https://tunewsinternational.com'
+
+export const userAgent = `Integreat Native App (${Platform.OS} ${NativeConstants.appVersion})`
```

Comment:
If this was discussed with/wished by @svenseeberg ignore this.
Our naming is normally `integreat app native`, at least in terms of mattermost, intellij project and repo docu so we could use that order here as well:
```suggestion
export const userAgent = `Integreat App Native (${Platform.OS} ${NativeConstants.appVersion})`
```
But I don't really mind, your choice :)

Reply:
But way more important: It should not be a hardcoded `Integreat` but instead use the build config name/app name.

Reply:
Usually a useragent is of the form `Name/Version (Kommentar)` https://de.wikipedia.org/wiki/User_Agent

```suggestion
export const userAgent = `Integreat/${NativeConstants.appVersion} (${Platform.OS})`
```

no need to add native imo as we know it from the OS.

Reply:
But it would also be android if we open the web app on android right, if we also set the user agent there. I'll ask @svenseeberg what he prefers.